examples/resources/b1ddi_dns_record/resource.tf

```diff
@@ -1,7 +1,7 @@
 terraform {
   required_providers {
     b1ddi = {
-      version = "0.1"
+      version = "0.1.5"
```

Comment:
You remove `version`. By default will get the latest version if not specified 
```suggestion
```

Reply:
Fixed

Reply:
formatting was done in all the files